index.html

```diff
@@ -8,5 +8,6 @@
     </head>
     <body>
         <h1>Mikel wuz here</h1>
+        <h4>Hey! Rohini wuz here!!</h4>
```

Comment:
👍 @Hanna-N9 @rohinij15 @aggarwalsshreya Just fyi this view in GitHub is where we will do our code review!